deployment/helm/image-metadata-orchestrator/templates/artifacts.yml

```diff
@@ -15,3 +15,11 @@ data:
       bucket: {{ required "A bucket for temporary data is required." .Values.storage.s3.bucket }}
       insecure: {{ .Values.storage.s3.insecure }}
       useSDKCreds: {{ .Values.storage.s3.useSDKCreds }}
+{{- if .Values.isLocal }}
```

Comment:
@sleyer is this a good way?

Reply:
I'd say if it can be avoided, it should be. But sometimes it just can't.